Dockerfile

```
@@ -0,0 +1,39 @@
+FROM golang:1.11-alpine3.8 as builder
+
+RUN apk add --no-cache --virtual .build-deps gcc musl-dev make git expat
```

```suggestion
RUN apk add --no-cache gcc musl-dev make git
```
- `expat` is part of musl-dev and I believe `gcc` is not needed
- `--virtual .build-deps` not needed. We only add it if we want to clean the whole virtual package later, which we don't.

---

Ok